### ISSUE TITLE
Editor: Overlapping Parent Page Selections

### ISSUE THREAD

**Issue by chad1008** [Automattic/wp-calypso#16999]
#### Steps to reproduce

Note: this is specific to Jetpack sites and WordPress.com Atomic sites
1. Starting at URL: WordPress.com/page
2. Create at least nine pages (enough that the parent page selector becomes a scrollable list)
3. Add an additional page, open Page Settings > Page Attributes and set a parent page - choose a parent from the middle of the list, not the bottom. We want there to be list elements below this child
4. Publish the child page
5. Open the editor (either on an existing top level page, or by clicking Add) and once again navigate to Page Settings > Page Attributes
6. Scroll to find the child page you pubilshed in steps 3-4

#### What I expected
The child page(s) should be clearly displayed below their parents, indented with each level

#### What happened instead
The indent is as expected, but the next top level page floats upward, overlapping the child page(s) above it.

#### Browser / OS version
FF 55.0 (64-bit)
Chrome Version 60.0.3112.90 (Official Build) (64-bit)
MacOS Sierra 10.12.6

#### Screenshot / Video
![screen shot 2017-08-08 at 11 23 46 am](https://user-images.githubusercontent.com/13856531/29080882-04aaffda-7c2f-11e7-87f6-82a7adcf9e8a.png)

 #user-report 
563298-hc

**Comment by spncrb** [comment]
This same issue looks to be happening with the link button too:
![screen shot 2017-08-11 at 12 23 06 pm](https://user-images.githubusercontent.com/8921630/29228978-e0a3066e-7e90-11e7-9b07-bc992890b5b2.png)

Same general scenario - lots of posts/pages - an AT site.

It is very jumpy and hard to follow/navigate.

Reproducible in chrome and safari

#user-report 
571404-hc

**Comment by rachelmcr** [comment]
The issue with adding a link is related to long post titles and is being tracked here: https://github.com/Automattic/wp-calypso/issues/15809

**Comment by chad1008** [comment]
Another report of the overlapping parent pages in 647902-hc

They're using WP Admin as a workaround, but it's a bit of a pain point for them so they're following up for updates.

**Comment by suyogyashukla** [comment]
I had a user asking about ETA of the fix on HappyChat: https://mc.a8c.com/support-stats/happychat/transcript.php?id=661108

**Comment by lsinger** [comment]
Also see #18123, @marekhrabe is ON IT 👍

**Comment by guicmazeredo** [comment]
I had a user today with the same issue. HC: [298978](https://mc.a8c.com/support-stats/happychat/transcript.php?id=298978)

Here is a GIF I made to show the issue: https://cl.ly/3z3k312O1O0Q

**Comment by marekhrabe** [comment]
This issue should be now fixed in production. Could we please ping affected users? :)
